repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/media/MimeType.java

```diff
@@ -73,17 +75,30 @@ public static MimeType guessMediaTypeFromString(String mimeType) {
                     return ct;
                 }
             }
-
-            // this is unreachable code, and at this point I'm too afraid to fix it
-            for (MimeType ct : values()) {
```

Comment:
None of the unit tests broke, and this really was unreachable code... so I removed it. Hopefully no other tests break, either.